spec/FilesController.spec.js

```diff
@@ -21,11 +21,11 @@ const mockAdapter = {
 
 // Small additional tests to improve overall coverage
 describe('FilesController', () => {
-  it('should properly expand objects', done => {
+  it('should properly expand objects', async done => {
```

Comment:
Could you keep the sync test and add the async test as an addition test? We need to test whether both work.

Reply:
Yeah, sure! I've added a new test to check getFileLocation for both the async and sync functions. I believe I've also fixed the errors that were causing the CI failures. Could you please review it and let me know if everything looks good now?